FlowCryptUI/Cell Nodes/ThreadMessageDraftCellNode.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ThreadMessageDraftCellNode.swift
+//  FlowCryptUI
+//
+//  Created by Ioan Moldovan on 8/14/23
+//  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
+//
+
+import AsyncDisplayKit
+import LetterAvatarKit
+import UIKit
+
+public final class ThreadMessageDraftCellNode: CellNode {
+
+    private lazy var avatarNode: ASImageNode = getAvatarImage(text: sender)
```

Comment:
Can we show some SF icon instead of letters in draft, to better differentiate it?

Example from Gmail app:

<img src="https://github.com/FlowCrypt/flowcrypt-ios/assets/1062093/99055c07-53bc-4d72-83b2-909bf404f58c" width="240">

Reply:
Sounds good!

Reply:
let's try `envelope.open` or `envelope.open.fill`, as it'll better show meaning of draft (which is not sent yet, so envelope is open)

Reply:
![image](https://github.com/FlowCrypt/flowcrypt-ios/assets/77344191/a7f5ada7-dac0-4c0a-8151-c9d969f33737)

Reply:
![image](https://github.com/FlowCrypt/flowcrypt-ios/assets/77344191/6706b734-4a08-4d56-8755-5749a140c8d3)
Like this?

Reply:
Yeah, this one looks good 👍

Reply:
👍